src/components/roadmapList/RoadmapList.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import style from './RoadmapList.module.css';
+import { getRoadmapList } from 'src/api';
+
+import Button from 'components/ui/Button';
+import Icon from 'components/ui/Icon';
+import Container from 'components/ui/Container';
+import { RoadmapListItem } from 'src/types';
+import Pagination from './Pagination';
+
+export const RoadmapList: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState<number>(1);
+  const [totalPage, setTotalPage] = useState<number>(1);
+  const [list, setList] = useState<RoadmapListItem[]>([]);
+  useEffect(() => {
+    getRoadmapList(currentPage).then(({ data }) => {
+      setList(data.results);
+      setTotalPage(Math.ceil(data.count / 6));
+    });
+  }, [currentPage]);
+
+  const handlePageChange = (value: number) => {
+    setCurrentPage(value);
+  };
+
+  return (
+    <div>
+      <div className={style.filter_wrapper}>
+        <Container>
+          <p className={style.filter_title}>Filters</p>
+          <div className={style.filter_box}>
+            <div className={style.filter_list}>
+              <p className={style.filter}>Category</p>
+              <p className={style.filter}>Level</p>
+              <p className={style.filter}>Local domain</p>
+            </div>
+            <Button variant="primary" size="lg">
+              Search
+            </Button>
+          </div>
+        </Container>
+      </div>
+      <Container>
+        <div className={style.list_wrapper}>
+          {list.length &&
+            list.map((el, ind) => (
+              <div key={`${el.pk}+${ind}`} className={style.list_item}>
+                <div className={style.item_header}>
+                  <p className={style.item_headerTitle}>{el.title}</p>
+                  <Icon
+                    id="icon-open"
+                    stroke="#2E3249"
+                    fill="#fff"
+                    width={28}
+                    height={28}
+                    boxStyles={style.pointer}
+                  />
+                </div>
+                <ul className={style.tags_list}>
+                  {el.tags.map((el, ind) => (
+                    <>
+                      <li key={`${el}+${ind}`} className={style.tags}>
+                        {el}
+                      </li>
+                    </>
+                  ))}
+                </ul>
+              </div>
+            ))}
+        </div>
+        <div className={style.footer}>
+          <p className={style.info}>
+            <a href="/login" className={style.link}>
```

Comment:
we have a <Link> react-router-dom component for that, and also yo can style it with the help of buttonVariant func, it'll cut on your code

Reply:
Using the button tag is not correct for the link

Reply:
it isn't a button tag, this is a only button styles like if you want to use some button variant on tags, which shouldn't be a button